README.md

```diff
@@ -1,3 +1,8 @@
+---
+cover: .gitbook/assets/Screenshot from 2023-07-27 17-22-02.png
+coverY: 0
+---
+
```

Comment:
That cover image looks real bad. Not our font

Reply:
I don't see how it's real bad. They are the same for docs main, boltz-backend, boltz-lnd and webapp. And it's the font we use for marketing stuff like https://twitter.com/Boltzhq/header_photo.